src/graph_compute/chromatic_scheduling.h

```diff
@@ -0,0 +1,105 @@
+#ifndef CHROMATIC_SCHEDULING_H_
+#define CHROMATIC_SCHEDULING_H_
+
+#if D1_CHROM
+
+#include <algorithm>
+#include "./common.h"
+
+struct scheddata_t {
+  vid_t cntColors;
+  vid_t * cntNodesPerColor;
+  vid_t * nodesByColor;
+};
+typedef struct scheddata_t scheddata_t;
+
+struct sched_t {
+  vid_t color;
+};
+typedef struct sched_t sched_t;
+
+#include "./update_function.h"
+
+static inline vid_t colorGraph(vertex_t * const nodes,
```

Comment:
Would be nice to have some explanatory comments here. I think I understand what this is doing, but still.